packages/editor-ui/src/components/mixins/executionsHelpers.ts

```diff
@@ -0,0 +1,70 @@
+import { IExecutionsSummary } from "@/Interface";
+import dateFormat from "dateformat";
+import mixins from "vue-typed-mixins";
+import { genericHelpers } from "./genericHelpers";
+
+export interface IExecutionUIData {
+	name: string;
+	label: string;
+	startTime: string;
+	runningTime: string;
+}
+
+export const executionHelpers = mixins(genericHelpers).extend({
```

Comment:
not a fan of adding more mixins...

Reply:
Me neither, but this logic is used in different components in executions view, so decided to externalize it like this.